### PR TITLE
Pretty print role list

### PR DESCRIPTION
**Pull request by sylveon**

### CODE REVIEW

File: Modix/Modules/UserInfoModule.cs [hunk 1]
```diff
@@ -69,7 +69,25 @@ public UserInfoModule(ILogger<UserInfoModule> logger)
...
+                            }
+                            else if (roles.Length > 1 && i + 1 == roles.Length)
+                            {
+                                separator = " and ";
+                            }
+                            else
+                            {
+                                separator = ", ";
+                            }
+                            builder.Append($"{separator}<@&{roles[i].Id}>");
```
Comment (JakenVeina) [inline Modix/Modules/UserInfoModule.cs]: You should ternary this bitch up.
```csharp
var separator = (i == 0) ? string.Empty
    : ((roles.Length > 1) && ((i + 1) == roles.Length)) ? ", and ";
    : ", ";
```

Comment (Cisien) [reply Modix/Modules/UserInfoModule.cs]: `roles[i].Mention`

Comment (JakenVeina) [reply Modix/Modules/UserInfoModule.cs]: A little.

Declaring a variable then assigning its value later just makes me cringe, cause it's about 90% of the time a good use for a functional syntax assignment, of some sort.

What you could do is move that to a local function, so you can make use of if/return syntax.
```csharp
if(...)
    return ...;
else if(...)
    return ...:
else
    return ...;
```

Or just leave it as is. Not a big deal.

Supposedly, we might get a switch/case style expression soon, that would work well here. E.G.
```csharp
var separator = switch
{
    case (i == 0):                                        string.Empty;
    case (roles.Length > 2) && (i == (roles.Length - 1)): ", and ";
    case (roles.Length > 1) && (i == (roles.Length - 1)): " and ";
    default:                                              ", ";
}
```

File: Modix/Modules/UserInfoModule.cs [hunk 2]
```diff
@@ -69,7 +69,25 @@ public UserInfoModule(ILogger<UserInfoModule> logger)
...
+                        for (int i = 0; i < roles.Length; i++)
+                        {
+                            string separator;
+                            if (i == 0)
+                            {
+                                separator = string.Empty;
+                            }
+                            else if (roles.Length > 1 && i + 1 == roles.Length)
+                            {
+                                separator = " and ";
```
[the remark this answers is not in the document]
Comment (sylveon) [reply Modix/Modules/UserInfoModule.cs]: No Oxford comma > Oxford comma

[the remark this answers is not in the document]
Comment (JakenVeina) [reply Modix/Modules/UserInfoModule.cs]: http://3.bp.blogspot.com/-yleEs15g51w/TlrawTwZSrI/AAAAAAAABQY/gnyx4k2pzcQ/s400/tommy-lee-jones-implied-face-palm.png

[the remark this answers is not in the document]
Comment (Trinitek) [reply Modix/Modules/UserInfoModule.cs]: +1 for Oxford comma

**Pull request merged**